frontend/src/components/activity/ScheduleEntry.vue

```diff
@@ -91,6 +93,14 @@ Displays a single scheduleEntry
         <template v-else>{{ $tc('global.button.back') }}</template>
       </v-btn>
 
+      <v-btn text icon class="d-hidden d-md-block" @click="togglePaperSize">
```

Comment:
i would hide this button on mobile.
(or on any device with a screen width <= 25cm)

Reply:
I used an inexisting utility class